lib/utils.ts

```diff
@@ -38,13 +38,6 @@ export const fetcher = async (url: string) => {
   return res.json();
 };
 
-export function getLocalStorage(key: string) {
```

Comment:
FYI - this wasn't used anymore, hence, I've removed it!

Also, as ai-chatbot uses `usehooks-ts` as a dependency, we could use its local storage hook instead if we need it again in the future: https://usehooks-ts.com/react-hook/use-local-storage